dashboard/app/models/standard.rb

```diff
@@ -7,9 +7,14 @@
 #  organization_id :string(255)      not null
 #  description     :text(65535)
 #  concept         :text(65535)
+#  category_id     :bigint
+#  framework_id    :integer
```

Comment:
Why is category_Id a bigint and framework_id an integer?

Reply:
category_id is a bigint because that is what is generated by `t.references`. `framework_id` could be a bigint too and that's probably the best practice for any ids pointing to another table. However, since we only expect to have 10s of them and `:integer` will be good for up to 2,000,000,000 frameworks, I don't think we'll run into any problems with framework_id being `:integer`.